github/client.go

```diff
@@ -544,14 +551,20 @@ func (client *Client) FindOrCreateToken(user, password, twoFactorCode string) (t
 	return
 }
 
-func (client *Client) api() (c *octokit.Client, err error) {
+func (client *Client) ensureAccessToken() (err error) {
 	if client.Host.AccessToken == "" {
-		host, e := CurrentConfig().PromptForHost(client.Host.Host)
-		if e != nil {
-			err = e
-			return
+		host, err := CurrentConfig().PromptForHost(client.Host.Host)
```

Comment:
Actually I just tested it now and it seems OK. The empty `return`s will correctly return the `err` object even if it has been "redefined" by `:=` operator. We're using the construct quite a lot throughout the codebase and it seems OK. I will keep it like this.